src/explorer/siteNodeBase.ts

```diff
@@ -203,16 +203,16 @@ export class SiteNodeBase extends NodeBase {
                 if (input === 'Install') {
                     opn(`https://git-scm.com/downloads`);
                 }
-                throw err;
+                throw new GitNotInstalledError();
```

Comment:
Can you move the `showErrorMessage()` and git url logic either into `GitNotInstalledError` or directly into `initAsyncCommand`? (If you put it in `GitNotInstalledError`, I imagine it would be in a 'prompt' or 'showMessage' function that gets called by `initAsyncCommand`)

That way we could potentially throw this `GitNotInstalledError()` in other places and it's still handled the same way